### PR TITLE
refactor: use Wallet lib instead of getUTXO

### PR DESCRIPTION
**Pull request by Alex-Werner**
## Issue being fixed or feature implemented

In order to remove getUTXO, and have a fully functional coverage, we switch from getUTXO to wallet into our test-suite.

## What was done?
- modified funding logic from individual components to SDK
- removed getUTXO
- removed fundAddress (for fundAccount)

## How Has This Been Tested?

This is actually some tests modifications.

## Breaking Changes
N/A

## Checklist:

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have added or updated relevant unit/integration/functional/e2e tests
- [X] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [X] I have assigned this pull request to a milestone

### CODE REVIEW

File: lib/test/createClientWithFundedWallet.js [hunk 1]
```diff
@@ -1,14 +1,7 @@
...
-} = require('@dashevo/dashcore-lib');
-
 const Dash = require('dash');
 
 const getDAPISeeds = require('./getDAPISeeds');
-
-const fundAddress = require('./fundAddress');
-const wait = require('../wait');
-
+const fundAccount = require('./fundAccount');
```
Comment (shumkov) [inline lib/test/createClientWithFundedWallet.js:4]: We already have `fundWallet` function in Wallet Lib. Let's just reuse it? https://github.com/dashevo/wallet-lib/pull/119/files#diff-1c8a94b728d0cccd2708ade2269dd0c6

**Review by shumkov — CHANGES_REQUESTED** [review]

File: lib/test/createClientWithFundedWallet.js (hunk 1 above)
Comment (Alex-Werner) [reply lib/test/createClientWithFundedWallet.js:4]: Yeah, I've saw that, but it's in wallet-lib. 
Here we are dependant on `dash` repo.

Comment (shumkov) [reply lib/test/createClientWithFundedWallet.js:4]: Yeah, we defo don't want to duplicate this logic everywhere, so I was thinking that we can include wallet-lib (that branch atm) into dependencies and reuse the `fundWallet` function.

Comment (Alex-Werner) [reply lib/test/createClientWithFundedWallet.js]: This is done, we now will need to wait the PR (in WLib)

**Review by shumkov — APPROVED** [review]
LGTM! 👍

**Pull request merged**